pages.fr/linux/btrfs-check.md

```diff
@@ -0,0 +1,32 @@
+# btrfs check
+
+> Vérifier l'état, ou réparer un système de fichier de type BTRFS.
+> Plus d'informations : <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-check>.
+
+- Vérifier l'état d'un système de fichier btrfs :
```

Comment:
```suggestion
- Vérifier l'état d'un système de fichier btrfs :
```
all line with colon should have this specific space

Reply:
I'm not sure what is the difference with the space character I used vs. the one you replaced in this commit but it clearly shows as being different. I re-used it for other items on the page. Thanks !

Reply:
Yes, this one in a "special space character" called "espace insécable" it is a special typo in French language and need to be pushed next (before or after) to several characters such as ":", "?" for instance. It means it can't be split when there is a carriage return. If you want, you can copy it from "Libreoffice Writer" to Github for example.